packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -525,7 +526,11 @@ export class LocalIntervalCollection<TInterval extends ISerializableInterval> {
                 interval.properties[MergeTree.reservedRangeLabelsKey] = [this.label];
             }
             if (interval.properties[reservedIntervalIdKey] === undefined) {
-                interval.properties[reservedIntervalIdKey] = uuid();
+                // Create a new ID and make it immutable.
+                Object.defineProperty(interval.properties, reservedIntervalIdKey, {
```

Comment:
what about the load case?

Reply:
I'm thinking, since we always make the ID immutable when we create it locally, the serialized form of an interval that's in a collection should have an immutable ID. So that should apply to serialized intervals that we load. I'll confirm in the dehydrate/rehydrate test.

Reply:
No, I'm wrong, serialization doesn't preserve writable, configurable, etc., so this affects both the load case and the remote add op case. I'll fix it.